code/modules/jobs/job_types/clown.dm

```diff
@@ -9,6 +9,7 @@
 	supervisors = "the head of personnel"
 	selection_color = "#dddddd"
 	chat_color = "#FF83D7"
+	special_notice = "There is a difference between harmless pranks and griefing. Know it."
```

Comment:
OOC

Reply:
> OOC

I don't see how that's bad.

Reply:
There's no problem with them being OOC. These are pretty much just policy reminders.

Reply:
Ehh, it still feels pretty hostile. I don't think there would be anything wrong with making it more friendly and IC